### ISSUE TITLE
Php parsing issue.

### ISSUE THREAD

**Issue by 0b1kn00b** [jdegoes/stax#1]
Parse error: syntax error, unexpected '(' in D:\stack\projects\PhpStax\bin\lib\Function2Extensions.class.php on line 207;

minimal case: 

import Prelude;
import PreludeExtensions;

using Prelude;
using PreludeExtensions;

class Main {

```
static function main() {
    var f : Function1<Dynamic,Dynamic> = function (y) {
        trace("ok");
    }
    f.uncurry();
}
```

}

**Comment by fponticelli** [comment]
Yes, I am aware that php is not very stax friendly. Running the tests I've been able to spot 2 or 3 minor bugs (in term of codes refactoring) that are enough to make php explode. I've reported on the haxe issue list and I will work on them asap. Thank you for reporting anyway, I'll try to add a specific test for your case too.

**Comment by fponticelli** [comment]
Can you confirm that this issue is still alive after haxe 2.08?
